src/dlgTriggerEditor.cpp

```diff
@@ -182,9 +187,26 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     QSizePolicy sizePolicy5(QSizePolicy::Expanding, QSizePolicy::Expanding);
     mpSourceEditorArea->setSizePolicy( sizePolicy5 );
     pVB1->addWidget( mpSourceEditorArea );
-    QPlainTextEdit * editp = mpSourceEditorArea->editor;
-    editp->setWordWrapMode( QTextOption::NoWrap );
-    connect(editp,SIGNAL(cursorPositionChanged()), this, SLOT(slot_cursorPositionChanged()));
+    mpSourceEditor = mpSourceEditorArea->editor;
+    mpSourceEditor->setWordWrapMode( QTextOption::NoWrap );
+#if QT_VERSION >= 0x050200
```

Comment:
Well the Linux build was (is) using Qt 5.0.2 (not entirely for this reason - the Ubuntu 12.04 LTS provides it out of the box) - but it is a useful catch for features that are not in all 5.x series and that is as early as we "Officially" go IIRC.

Good catch though - let me update that to: `#if QT_VERSION >= 0x050300`!